sdk/python/feast/repo_config.py

```diff
@@ -165,6 +165,9 @@ class RepoConfig(FeastBaseModel):
     coerce_tz_aware: Optional[bool] = True
     """ If True, coerces entity_df timestamp columns to be timezone aware (to UTC by default). """
 
+    ignore_infra_changes: bool = False
+    """ Signals whether to apply infrastructure changes during Registry creation """
```

Comment:
when would one want to use this flag? mind adding brief details here?

Reply:
@dominicbecker 
We need to support the following: infrastructure changes should only be applied once during a deploy. With every `FeatureStore` instance, the registry is created from scratch. Without extra information, there's no indication when or where we should apply infrastructure changes.

Reply:
do you mind updating the docstring with some more info?

e.g. include that it only applies for `registry_type: memory`

Reply:
Updated